### ISSUE TITLE
Modernize RadioButton for Accessibility

### ISSUE THREAD

**Issue by marc-bouvier** [iroco-co/iroco-ui#17]
The following can be improved

- focus is not visible (https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#10.7)
- hidden radio input could be with `appearance:none` (https://caniuse.com/css-appearance)
- cannot be used with keyboard

## Ressources

https://moderncss.dev/pure-css-custom-styled-radio-buttons/
